tutorials/prep.md

```diff
@@ -19,11 +19,50 @@ Finally, PrEP usage is updated for anyone starting, continuing, stopping, or res
 
 Individuals can be specifically tested to start PrEP for the first time, but tested people in the general population can also decide to start PrEP. When people continue PrEP usage, they can either continue with their current PrEP or switch to a different type if their favoured PrEP has changed. PrEP usage can be stopped for two reasons – an individual can choose to stop, or they can become ineligible for PrEP. People who have chosen to stop taking PrEP but are still eligible can also choose to restart, but anyone who stopped taking PrEP due to a break in eligibility automatically restarts.
 
+### PrEP Columns
+
+- *`R_PREP`* - A semi-permanent random float variable that determines whether an individual is risk informed or suspects they are at risk enough to take PrEP. Rerolled only for ineligible people when determining PrEP eligibility each time step.
+- *`PREP_ORAL_PREF`* - A float value between [0, 1] drawn from a beta distribution that determines and individual's preference for oral PrEP.
+- *`PREP_CAB_PREF`* - A float value between [0, 1] drawn from a beta distribution that determines and individual's preference for cabotegravir PrEP.
+- *`PREP_LEN_PREF`* - A float value between [0, 1] drawn from a beta distribution that determines and individual's preference for lenacapavir PrEP.
+- *`PREP_VR_PREF`* - A float value between [0, 1] drawn from a beta distribution that determines and individual's preference for vaginal ring PrEP.
```

Comment:
About these intervals – I'm not actually sure whether they're [0, 1] or (1, 0). The docs don't say, and the referenced beta distribution Wikipedia page states the interval can be [0, 1] or (1, 0), so that's not very helpful.

Reply:
Current program => [0, 1) from numpy RNG